docs/pages/database-access/guides/mongodb-self-hosted.mdx

```diff
@@ -0,0 +1,348 @@
+---
+title: Database Access with Self-Hosted MongoDB
+description: How to configure Teleport Database Access with self-hosted MongoDB
+---
+
+# Self-Hosted MongoDB
+
+<Admonition type="note" title="Supported versions">
+  Teleport Database Access supports MongoDB 3.6 and higher. Older versions have
+  not been tested and are not guaranteed to work. MongoDB 3.6 was released in
+  November 2017 and reached EOL in April 2021 so if you're still using an older
+  version, consider upgrading.
+</Admonition>
+
+## Create Certificate/Key Pair
+
+(!docs/pages/includes/database-access/tctl-auth-sign.mdx!)
+
+Create the secrets:
+
+<Tabs>
+  <TabItem label="Standalone">
+  When connecting to standalone MongoDB, sign certificate for the hostname over
```

Comment:
May want to double-check this component. Looks like a tag is unclosed.

Reply:
It is closed, renders fine also.